src/main/java/com/amazon/ion/impl/_Private_IonTextWriterBuilder.java

```diff
@@ -1,48 +1,20 @@
-/*
- * Copyright 2007-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
 package com.amazon.ion.impl;
 
-import static com.amazon.ion.impl._Private_Utils.initialSymtab;
 
-import com.amazon.ion.IonCatalog;
-import com.amazon.ion.IonSystem;
-import com.amazon.ion.IonWriter;
-import com.amazon.ion.SymbolTable;
-import com.amazon.ion.system.IonSystemBuilder;
 import com.amazon.ion.system.IonTextWriterBuilder;
 import com.amazon.ion.system.SimpleCatalog;
-import com.amazon.ion.util._Private_FastAppendable;
-import java.io.OutputStream;
 
 /**
+ * Contains configuration common to text writers for all Ion versions.
  * NOT FOR APPLICATION USE!
  */
-public class _Private_IonTextWriterBuilder
+public abstract class _Private_IonTextWriterBuilder
```

Comment:
Extended by `_Private_IonTextWriterBuilder_1_0` and `_Private_IonTextWriterBuilder_1_1`.